packages/template-retail-react-app/app/components/responsive/index.jsx

```diff
@@ -11,19 +11,33 @@ import {Box} from '@salesforce/retail-react-app/app/components/shared/ui'
 /**
  * Render the children in the DOM but visually hide them on desktop
  * @param children - isomorphic components used within a responsive design
+ * @param rest - additional props to be passed to the Box component
  */
-export const HideOnDesktop = ({children}) => (
-    <Box display={{base: 'block', lg: 'none'}}>{children}</Box>
+export const HideOnDesktop = ({children, ...rest}) => (
+    <Box display={{base: 'block', lg: 'none'}} {...rest}>
+        {children}
+    </Box>
 )
-HideOnDesktop.propTypes = {children: PropTypes.node}
+HideOnDesktop.propTypes = {
+    children: PropTypes.node,
+    display: PropTypes.object,
+    rest: PropTypes.object
+}
 
 /**
  * Render the children in the DOM but visually hide them on mobile
  * @param children - isomorphic components used within a responsive design
+ * @param rest - additional props to be passed to the Box component
  */
-export const HideOnMobile = ({children}) => (
-    <Box display={{base: 'none', lg: 'block'}}>{children}</Box>
+export const HideOnMobile = ({children, ...rest}) => (
+    <Box display={{base: 'none', lg: 'block'}} {...rest}>
+        {children}
+    </Box>
 )
-HideOnMobile.propTypes = {children: PropTypes.node}
+HideOnMobile.propTypes = {
+    children: PropTypes.node,
+    display: PropTypes.object,
+    rest: PropTypes.object
```

Comment:
We don't need to specify the prop type for this `rest` because we won't be able to predict what they are. The linter won't expect it either.